### DISCUSSION TITLE
ASC712 Support issue

### DISCUSSION THREAD

**Discussion by Sayed-afridy** [arendst/Tasmota#19379] [Q&A]
how can i  integrate ACS712 support into the xsns_02_analog.ino driver for the analog input as an additional mode for the ADC Input.

**Comment by barbudor** [comment]
Why an additional mode?
ACS712 is fully compatible with CT input type

**Reply by barbudor** [reply]
https://github.com/arendst/Tasmota/discussions/18172#discussioncomment-5572008
